src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -1211,14 +1211,14 @@ bool SystemDictionaryShared::add_unregistered_class(InstanceKlass* k, TRAPS) {
 }
 
 // This function is called to resolve the super/interfaces of shared classes for
-// non-built-in loaders. E.g., ChildClass in the below example
+// non-built-in loaders. E.g., KlassClass in the below example
 // where "super:" (and optionally "interface:") have been specified.
 //
 // java/lang/Object id: 0
 // Interface   id: 2 super: 0 source: cust.jar
-// ChildClass  id: 4 super: 0 interfaces: 2 source: cust.jar
+// KlassClass  id: 4 super: 0 interfaces: 2 source: cust.jar
```

Comment:
`KlassClass` sounds like it's a "class of classes" which would be misleading in this context. How about `MyClass` instead?

Reply:
How about SharedClass ?